### ISSUE TITLE
Gettext warnings because some translation files specify nplurals without rules

### ISSUE THREAD

**Issue by Sinc63** [bitwalker/timex#734]
### Steps to reproduce
timex version 3.7.9
gettext version 0.22.0
mix deps.compile timex

### Description of issue

- What are the expected results?
Clean compile

- Actual result: numerous messages like:
```
warning: Plural-Forms headers in the form "nplurals=<int>" (without the "plural=<rule>" part following) are invalid and support for them will be removed in future Gettext versions. Make sure to use a complete Plural-Forms header, which also specifies the pluralization rules, or remove the Plural-Forms header completely. If you do the latter, Gettext will use its built-in pluralization rules for the languages it knows about (see Gettext.Plural).
For the "bg" locale, you can use the following header:

nplurals=2; plural=(n != 1);

  (gettext 0.22.0) lib/gettext/plural.ex:288: Gettext.Plural.init/1
  (gettext 0.22.0) lib/gettext/compiler.ex:571: Gettext.Compiler.compile_plural_forms/4
  (gettext 0.22.0) lib/gettext/compiler.ex:509: Gettext.Compiler.compile_po_file/5
  (gettext 0.22.0) lib/gettext/compiler.ex:450: Gettext.Compiler.compile_unified_po_file/4
  (elixir 1.13.3) lib/enum.ex:1593: Enum."-map/2-lists^map/1-0-"/2
  (elixir 1.13.3) lib/enum.ex:1593: Enum."-map/2-lists^map/1-0-"/2

warning: Plural-Forms headers in the form "nplurals=<int>" (without the "plural=<rule>" part following) are invalid and support for them will be removed in future Gettext versions. Make sure to use a complete Plural-Forms header, which also specifies the pluralization rules, or remove the Plural-Forms header completely. If you do the latter, Gettext will use its built-in pluralization rules for the languages it knows about (see Gettext.Plural).
For the "bg" locale, you can use the following header:

nplurals=2; plural=(n != 1);

  (gettext 0.22.0) lib/gettext/plural.ex:288: Gettext.Plural.init/1
  (gettext 0.22.0) lib/gettext/compiler.ex:564: Gettext.Compiler.nplurals/3
  (gettext 0.22.0) lib/gettext/compiler.ex:510: Gettext.Compiler.compile_po_file/5
  (gettext 0.22.0) lib/gettext/compiler.ex:450: Gettext.Compiler.compile_unified_po_file/4
  (elixir 1.13.3) lib/enum.ex:1593: Enum."-map/2-lists^map/1-0-"/2
  (elixir 1.13.3) lib/enum.ex:1593: Enum."-map/2-lists^map/1-0-"/2
```

Errors are reported for bg, cs and he locales.
In one file I checked there are no plurals in the file so the nplurals line can be trivially deleted.

**Comment by Hermanverschooten** [comment]
There seems to be a PR open for this, #733

**Comment by JasonTrue** [comment]
Yes, my pr #733 now resolves this. There do seem to be (remaining) issues with the actual behavior of Russian and similar languages (we verified the problem reproduces with Croatian after testing with a native speaker on our team) so there may be additional work

**Comment by Sinc63** [comment]
I wrote the issue because I didn't see another one among issues, then found the PR. So yes, go ahead and tie this to the PR as well.

**Comment by bitwalker** [comment]
Fixed in 3.7.11
